app/classifier/index.cjsx

```diff
@@ -55,6 +58,13 @@ Classifier = React.createClass
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
 
+    @context.geordi.remember subjectID: @props.subject?.id
+
+  componentWillUnmount: () ->
+    try
+      @context.geordi?.forget ['subjectID']
+
```

Comment:
i figured if it fails we don't really care, we just don't want to throw an error to the app